.github/workflows/ci.yml

```diff
@@ -24,6 +24,12 @@ jobs:
     steps:
       - uses: actions/checkout@v4
 
+      - name: Markdown lint
```

Comment:
should be
```
      - uses: DavidAnson/markdownlint-cli2-action@v17
        with:
          globs: 'docs/**/*.md'
```

the one you're using here is archived and not maintained

Reply:
Done